code/game/sound.dm

```diff
@@ -68,11 +68,6 @@
 		if(below_turf && istransparentturf(turf_source))
 			listeners += SSmobs.clients_by_zlevel[below_turf.z]
 
-		for(var/mob/listening_mob as anything in listeners)
```

Comment:
I don't think you want to remove this

Reply:
https://github.com/tgstation/tgstation/pull/67989

Reply:
push upstream please